tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -1,52 +1,79 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 import Button from '../Button'
+import STTComboBox from '../STTComboBox'
 import { history } from '../../configureStore'
-import { setYear } from '../../actions/reports'
+import { setStt, setYear } from '../../actions/reports'
 
 /**
- * @param {string} selectedYear = The year that the user has chosen from the
- * Select component.
- *
  * Reports is the home page for users to file a report.
  * The user can select a year
  * for the report that they would like to upload and then click on
  * `Begin Report` to begin uploading files for that year.
  */
 function Reports() {
   const selectedYear = useSelector((state) => state.reports.year)
+  const selectedStt = useSelector((state) => state.reports.stt)
```

Comment:
It may be helpful to note somewhere here that `state` is referring to the Redux store state and not the same type of state from the STT model.

Reply:
Agreed